src/main/resources/hudson/maven/MavenModuleSet/index.jelly

```diff
@@ -42,26 +42,10 @@ THE SOFTWARE.
             ${act.displayName}
           </t:summary>
         </j:forEach>
-        <t:summary icon="folder.svg" href="ws/" permission="${it.WORKSPACE}">
```

Comment:
why removing access to workspace? this has nothing to do with duplicate link test